docs/service-catalog/docs/062-osba-instances.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Instances view
+type: UI Contracts
+---
+
+This document describes the mapping of [OSBA service objects](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#service-objects) in the Kyma Console Instances view.
```

Comment:
I like the idea of linking directly to OSBA, just please make sure you link properly
it is not always service object section
for example for catalog and instance view it is also plan object, in other cases also other objects, depending what we display

maybe better link here https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#catalog-management ? what do you think?